src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -85,6 +86,14 @@ public function getConfigTreeBuilder()
                 ->scalarNode('ide')->defaultNull()->end()
                 ->booleanNode('test')->end()
                 ->scalarNode('default_locale')->defaultValue('en')->end()
+                ->booleanNode('use_language_headers')
+                    ->info('Enables using the Accept-Language HTTP header to determine the locale and setting the Content-Language HTTP header.')
+                    ->defaultFalse()
+                ->end()
+                ->arrayNode('enabled_locales')
+                    ->info('A list of enabled locales. Allows to determine the user\'s locale according to the Accept-Language header.')
```

Comment:
this description is wrong. this list is not what determines the usage of `Accept-Language`. The `use_language_headers` option does that.

Reply:
should be better now